### DISCUSSION TITLE
준비큐와 대기큐가 포함된 프로세스 상태 다이어그램을 그려주세요!

### DISCUSSION THREAD

**Discussion by uzjaee** [OurStudyLog/operatingsystem#23] [Q&A] [answered]
11-1 퀴즈입니다. 
다이어그램을 그려주세요

**Comment by Sleepingoff** [comment, marked as answer]
![image](https://github.com/OurStudyLog/operatingsystem/assets/111721483/8cec36a8-eff3-45c5-b3ec-035758bc5275)

**Comment by ImSeon01** [comment]
<img width="853" alt="image" src="https://github.com/OurStudyLog/operatingsystem/assets/121650379/ca79a16a-68bd-4da7-b765-d44d4b358d5c">
